File: 0014-Longest-Common-Prefix/solution.java
import java.util.Arrays;

class Solution {
    public String longestCommonPrefix(String[] strs) {
        Arrays.sort(strs);
        String first = strs[0];
        String last = strs[strs.length - 1];
        int len = 0;
        for(int i = 0; i < strs[0].length(); i++){
            
            if(first.charAt(i) == last.charAt(i)){
                len++;
            } else{
                break;
            }
        }
        return first.substring(0, len);
    }
}

File: 0014-Longest-Common-Prefix/solution.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        strs.sort()
        first = strs[0]
        last = strs[-1]
        length = 0;

        for i in range(len(first)):
            if first[i] == last[i]:
                length += 1
            else:
                break
        return first[:length]